src/index.html

```diff
@@ -1,15 +1,176 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="POST"
+        onsubmit="onSubmit()"
+        >
+      <fieldset class="fiedset-block">
+        <legend>Personal information</legend>
+        <div class="fiedset-block__input">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
атрибуты должны иметь два отступа от тега
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
            required
          >
```

Reply:
как в моем комментарии
![image](https://user-images.githubusercontent.com/36257947/106723026-57e37880-660f-11eb-86b7-cb58d9900a20.png)